### ISSUE TITLE
[Governance] Showing wrong number of active proposals

### ISSUE THREAD

**Issue by thi4go** [decred/decrediton#3781]
It seems that we are showing the wrong number of active proposals on the Overview page. We currently have two ongoing votes, and one in discussion. 

This is what I'm getting on mainnet 1.7.3: 

<img width="525" alt="image" src="https://user-images.githubusercontent.com/2729122/176444367-15581984-7dd7-4273-be69-04783f1eaa4e.png">

**Comment by bgptr** [comment]
Yeah, maybe the naming could be improved. It indicates the number of active proposals that have not yet been voted by the user and requires the users' attention.
